php/libraries/NDB_Page.class.inc

```diff
@@ -280,6 +280,46 @@ class NDB_Page
     }
 
     /**
+     * Wrapper to create a radio button
+     *
+     * @param string $name       The field name of this radio button
+     * @param string $groupLabel The label to attach to this group of
+     *                           radio buttons
+     * @param array  $radios     The array of arrays describing radio
+     *                           elements to be added as a group. Ex:
+     *                           array('label'=>'Yes','value'=>'Y')
+     * @param array  $options    Options to pass to LorisForm for this
+     *                           radio button. Do not pass value here.
+     *
+     * @return none
+     */
+    function addRadio($name, $groupLabel, $radios, $options)
+    {
+        foreach ($radios as $radElement) {
+            if ($options === null) {
```

Comment:
if you set the default value of `$options` to `array()` in the function declaration you'll remove the need of this if statement